File: Challenge15/easy.dart
//Definition

//🙌🏼 Easy: I want you to write an algorithm that finds how old the user is by taking their date of birth.

import 'dart:io';

void main(List<String> args) {
  print("You are ${DateTime.now().year - getBirthyear()} years old");
}

int getBirthyear() {
  int? year;
  do {
    print("Enter your birtyear?");
    year = int.tryParse(stdin.readLineSync() ?? "");
  } while (year == null);

  return year;
}

File: Challenge13/easy.c
//Definition

//I want you to get a number from the user and tell the user if that number is prime or not.


#include <stdio.h>
void checkPrime(int N);

int main()
{
    int N = 546;
    checkPrime(N);
 
    return 0;
}

void checkPrime(int N){
    for (int i = 2; i <= N/2; i++)
    {
        if(N%i == 0){
            printf("The number %d is a Prime Number\n", N);
            return;
        }
    }
    printf("The number %d is not a Prime Number\n", N);
}